src/main/java/core/basesyntax/io/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.io;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String content, String fileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new java.io.FileWriter(fileName))) {
```

Comment:
It's recommended to use try-with-resources to ensure that the resources are closed properly. However, since `java.io.FileWriter` is also `AutoCloseable`, you should wrap it in the try-with-resources statement as well to ensure it's closed even if an exception occurs before the BufferedWriter is instantiated.